docs/_src/language/basic.md

```diff
@@ -60,48 +51,65 @@ explore 'malloy-data.faa.airports'
 ```
 
 ## Everything has a Name
-In Malloy, all output fields are named.  Malloy uses `<fieldname> is <expression>` instead of SQL's `<expression> as <fieldname>` so that the object being named comes first.  Having the output column name written first makes imagining the structure the output table easier.
 
-## Expressions are similar to SQL
-Mostly, your SQL expressions just work.  In Malloy, you can use almost all the same functions you can use in SQL (there are some new ones for convienece).
+In Malloy, all output fields are named. This means that any time a query includes a
+calculation or agregation, it must be aliased.
+
+```malloy
+--! {"isRunnable": true, "runMode": "auto", "isPaginationEnabled": true}
+explore 'malloy-data.faa.airports' | reduce
+  max_elevation is max(elevation)
+```
+
+Malloy uses `name is value` instead of SQL's `value as name` so the object being named comes first. Having the output column name written first makes imagining the structure the output table easier.
 
-The result of this is fairly straightforward based on a knowledge of SQL.
+Columns from a table and fields defined in an explore already have names, and can be referenced directly.
 
 ```malloy
 --! {"isRunnable": true, "runMode": "auto", "isPaginationEnabled": true}
-explore 'malloy-data.faa.airports'
-| reduce
-  state_and_county is concat(state,', ', county )
+explore 'malloy-data.faa.airports' | project
```

Comment:
Since we've already introduced `project` by this point, I figured it'd be good to mix things up, so it's clear these principles apply everywhere, not just in `reduce` statements...